<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PassPasswordsSettings</class>
 <widget class="QWidget" name="PassPasswordsSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>343</width>
    <height>204</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PassPasswords Settings</string>
  </property>
  <layout class="QVBoxLayout" name="vlayout_1">
   <item>
    <layout class="QHBoxLayout" name="hlayout_1">
     <item>
      <spacer name="hspace_1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_icon">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="pixmap">
        <pixmap resource="passpasswords.qrc">:/passpasswords/data/icon.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_title">
       <property name="text">
        <string>&lt;h3&gt;Pass Passwords&lt;/h3&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="hspace2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hlayout_2">
     <item>
      <spacer name="hspace_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="vlayout_2">
       <item>
        <widget class="QLabel" name="label_passdir">
         <property name="text">
          <string>Password storage directory:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="text_passdir">
        </widget>
       </item>

       <item>
        <widget class="QLabel" name="label_rootnode">
         <property name="text">
          <string>Only use passwords under this node (leave blank for all nodes):</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="text_rootnode">
        </widget>
       </item>

     <item>
      <widget class="QLabel" name="label_writenode">
       <property name="text">
        <string>Always store passwords under this node:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="text_writenode">
      </widget>
     </item>

      </layout>
     </item>
     <item>
      <spacer name="hspace_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="passpasswords.qrc"/>
 </resources>
 <connections/>
</ui>
